cmd/sops/main.go

```diff
@@ -1203,7 +1203,7 @@ func main() {
 					return toExitError(err)
 				}
 				if _, err := os.Stat(fileName); os.IsNotExist(err) {
-					return common.NewExitError("Error: cannot operate on non-existent file", codes.NoFileSpecified)
+					return common.NewExitError(fmt.Sprintf("Error: cannot operate on non-existent file [%s]", fileName), codes.NoFileSpecified)
```

Comment:
Why use `[]` to wrap the filename, instead of quotes?

Reply:
That's just a personal preference, things in `[]`s are usually easy to double-click and highlight in a terminal -- but I could definitely use quotes, is that preferred?

Reply:
Good question... I grepped a bit, and it looks like the common way is to use no delimiter, and separate it from the previous text by a colon, i.e. like this:
```suggestion
					return common.NewExitError(fmt.Sprintf("Error: cannot operate on non-existent file: %s", fileName), codes.NoFileSpecified)
```

Reply:
Ahhh, so that I think is a different pattern, I saw that and it seems to be more for when there's an internal error that should be surfaced. 

Since the file being non-existent *is* the "terminal" error, I put the file name directly in the message -- augmenting it rather than wrapping another error's message, essentially.

Reply:
I think in that case using `%q` instead of manually quoting is preferable.

Reply:
Sure -- will update to use `%q` everywhere `[%s]` was